crest/Assets/Crest/Crest/Scripts/Helpers/FloatingOrigin.cs

```diff
@@ -18,7 +18,9 @@
 // NOTE 2: Of particular note - any world space texturing should have a period that tiles perfectly across the teleport distance, otherwise visible
 // pops will occur. Example - set teleport radius to 16384m, and set normal map scale to 16m which divides evenly into the teleport radius.
 
+using System.Collections;
 using UnityEngine;
+using UnityEngine.Experimental.Rendering;
```

Comment:
does this work in our current supported versions? just making sure there isnt something that got moved out of Experimental

Reply:
It is experimental for all versions so far. I can surround it and dependent code with CREST_DEBUG for now? It is for the RenderDoc capture which I doubt anyone will need.

Reply:
After reading your other comment I will wrap with CREST_DEBUG as going to wrap that stuff too.

Reply:
Done. I have wrapped it with CREST_DEBUG and wrapped RenderDoc too. Thanks.